WMF Framework/Widget/Models/WidgetSettings.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public struct WidgetSettings: Codable {
+
+	// MARK: - Properties
+
+	public static let `default` = WidgetSettings(siteURL: URL(string: "https://en.wikipedia.org")!, languageCode: "en", languageVariantCode: nil)
```

Comment:
The intent of it was just to have some kind of sensible default displayed in widget (before the app has a chance to update it) instead of the worst case of no content for a new widget. In truth, we're already doing a bit of EN specific stuff like this in the other widgets. It's _possible_ it could have an unwanted EN display on the home screen widget (not in the app itself), but I think that'd be unlikely or temporary given any non-trivial use the app itself.

This particular setting is so transient – a refresh or language change in app will trigger it being corrected – that I think it's ok to leave it. But lemme know if you still feel strongly about wanting it to error out instead, and I can refactor the widget to do so.